VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-toastr.service.spec.ts

```diff
@@ -261,15 +261,11 @@ describe('NotificationToastrService', () => {
         toastrService.show.and.returnValue(mockToast);
 
         // Act
-        service.reportPoorConnection(
-            new ParticipantHeartbeat(globalConference.id, globalParticipant.id, HeartbeatHealth.Poor, '', '', '', '')
-        );
-        service.reportPoorConnection(
-            new ParticipantHeartbeat(globalConference.id, globalParticipant.id, HeartbeatHealth.Poor, '', '', '', '')
-        );
-        service.reportPoorConnection(
-            new ParticipantHeartbeat(globalConference.id, globalParticipant.id, HeartbeatHealth.Poor, '', '', '', '')
-        );
+        for (let i = 0; i < 26; i++) {
```

Comment:
can we test that 27 returns 2?

Reply:
also tied to the interval that may change or be unreliable, could we use a timer instead to reset? and use ticks(120000) in the test?

Reply:
not the case, should be one